tools/wheels/cibw_before_build.sh

```diff
@@ -26,40 +13,40 @@ elif [ -z $INSTALL_OPENBLAS ]; then
     export INSTALL_OPENBLAS=true
 fi
 
-# Install Openblas from scipy-openblas64
+# Install OpenBLAS from scipy-openblas32|64
 if [[ "$INSTALL_OPENBLAS" = "true" ]] ; then
-    # by default, use scipy-openblas64
+    # By default, use scipy-openblas64
+    # On 32-bit platforms and on win-arm64, use scipy-openblas32
     OPENBLAS=openblas64
-    # Possible values for RUNNER_ARCH in github are
-    # X86, X64, ARM, or ARM64
-    # TODO: should we detect a missing RUNNER_ARCH and use platform.machine()
-    #    when wheel build is run outside github?
-    # On 32-bit platforms, use scipy_openblas32
-    # On win-arm64 use scipy_openblas32
+    # Possible values for RUNNER_ARCH in GitHub Actions are: X86, X64, ARM, or ARM64
     if [[ $RUNNER_ARCH == "X86" || $RUNNER_ARCH == "ARM" ]] ; then
         OPENBLAS=openblas32
     elif [[ $RUNNER_ARCH == "ARM64" && $RUNNER_OS == "Windows" ]] ; then
         OPENBLAS=openblas32
     fi
-    echo PKG_CONFIG_PATH is $PKG_CONFIG_PATH, OPENBLAS is ${OPENBLAS}
-    PKG_CONFIG_PATH=$PROJECT_DIR/.openblas
-    rm -rf $PKG_CONFIG_PATH
-    mkdir -p $PKG_CONFIG_PATH
-    python -m pip install -r requirements/ci_requirements.txt
-    python -c "import scipy_${OPENBLAS}; print(scipy_${OPENBLAS}.get_pkg_config())" > $PKG_CONFIG_PATH/scipy-openblas.pc
-    # Copy the shared objects to a path under $PKG_CONFIG_PATH, the build
-    # will point $LD_LIBRARY_PATH there and then auditwheel/delocate-wheel will
-    # pull these into the wheel. Use python to avoid windows/posix problems
-    python <<EOF
+
+    # The PKG_CONFIG_PATH environment variable will be pointed to this path in
+    # pyproject.toml and .github/workflows/wheels.yml. Note that
+    # `pkgconf_path` here is only a bash variable local to this file.
+    pkgconf_path=$PROJECT_DIR/.openblas
+    echo pkgconf_path is $pkgconf_path, OPENBLAS is ${OPENBLAS}
+    rm -rf $pkgconf_path
+    mkdir -p $pkgconf_path
+    python -m pip install -r $PROJECT_DIR/requirements/ci_requirements.txt
+    python -c "import scipy_${OPENBLAS}; print(scipy_${OPENBLAS}.get_pkg_config())" > $pkgconf_path/scipy-openblas.pc
+
+    # Copy scipy-openblas DLL's to a fixed location so we can point delvewheel
+    # at it in `repair_windows.sh` (needed only on Windows because of the lack
+    # of RPATH support).
+    if [[ $RUNNER_OS == "Windows" ]]; then
+        python <<EOF
```

Comment:
This bit is a little complicated to follow for me because the indentation is changing.